.github/workflows/checklink.yaml

```diff
@@ -0,0 +1,14 @@
+name: Check Markdown links
+
+on: push
```

Comment:
It's a little bit late once the broken links get merged. How about running the link on each pull request.

Reply:
fixed